lexus_rx_450h_2019/drivers.launch.py

```diff
@@ -68,14 +68,14 @@ def generate_launch_description():
         ]
     )
 
-    dsrc_group = GroupAction(
-        condition=IfCondition(PythonExpression(["'dsrc_driver' in '", drivers, "'.split()"])),
+    v2x_driver_group = GroupAction(
+        condition=IfCondition(PythonExpression(["'v2x_ros_driver' in '", drivers, "'.split()"])),
         actions=[
             PushRosNamespace(EnvironmentVariable('CARMA_INTR_NS', default_value='hardware_interface')),
             IncludeLaunchDescription(
-                PythonLaunchDescriptionSource([ FindPackageShare('dsrc_driver'), '/launch/dsrc_driver.py']),
+                PythonLaunchDescriptionSource([ FindPackageShare('v2x_ros_driver'), '/launch/dsrc_driver.launch.py']),
```

Comment:
Need to change `/launch/dsrc_driver.launch.py` to `/launch/v2x_ros_driver.launch.py`